src/routes/events/index.svelte

```diff
@@ -37,7 +37,10 @@
 
 <Spacing --min="100px" --med="175px" --max="200px" />
 
-<h2 class="size-lg headers">Upcoming Events 📅</h2>
+<div class="main-header">
+  <h2 class="size-lg headers">Upcoming Events</h2>
+  <img src="assets/bluecalender.svg" alt="Blue Calender" />
```

Comment:
jk can u add a / in front of this and then re pull request LOL

Reply:
i dont think it's the most important thing to have but it's tehcnically correct idk thoughts? @EthanThatOneKid

Reply:
> i dont think it's the most important thing to have but it's tehcnically correct idk thoughts? @EthanThatOneKid

Since we will add the / in [the last PR](https://github.com/EthanThatOneKid/acmcsuf.com/pull/468#discussion_r872953338), we should probably add the / in this PR as well. 

> jk can u add a / in front of this and then re pull request LOL

A quick follow-up PR sounds good to me.